homeassistant/components/trafikverket_camera/coordinator.py

```diff
@@ -39,7 +39,9 @@ class CameraData:
 class TVDataUpdateCoordinator(DataUpdateCoordinator[CameraData]):
     """A Trafikverket Data Update Coordinator."""
 
-    def __init__(self, hass: HomeAssistant, entry: ConfigEntry) -> None:
+    config_entry: ConfigEntry
```

Comment:
```suggestion
    config_entry: TVCameraConfigEntry
```

To resolve the circular import, use
```py
if TYPE_CHECKING:
    from . import TVCameraConfigEntry
```